### PR TITLE
Added the D! Club, home of concerts, club nights and more

### PR DESCRIPTION
**Pull request by MarkJaroski**
I suppose I should filter it, but I just couldn't bring myself too since I'm interested in knowing about all of these classes of events.

Also, I tried extracting the date from the URL it seems that this doesn't work:

```
    location: ["20060102"]
```

### CODE REVIEW

File: croncert-config.yml [hunk 1]
```diff
@@ -699,6 +699,59 @@ scrapers:
...
+        type: date
+        components:
+          - covers:
+              day: true
+              month: true
+              year: true
+            location:
+              selector: a.main
+              attr: href
+              regex_extract:
```
Comment (jakopako) [inline croncert-config.yml:738]: You probably meant to do sth like this to extract the date from the url. Like this it works.

File: croncert-config.yml [hunk 2]
```diff
@@ -699,6 +699,59 @@ scrapers:
...
+      - name: style
+        type: text
+        location:
+          selector: div.info > div.details > p.style
+      - name: url
+        type: url
+        location:
+          selector: a.main
+          attr: href
+      - name: title
```
Comment (jakopako) [inline croncert-config.yml:723]: For the title we have the same problem that we discussed in the Dachstock issue. I opened an issue in goskyr for that: https://github.com/jakopako/goskyr/issues/188

Comment (jakopako) [reply croncert-config.yml:723]: Nevertheless I'm merging this config. We can update it once the new functionality is ready.

**Comment by jakopako** [comment]
It's fine to include the different categories. In the broadest sense they're all concerts :D

File: croncert-config.yml (hunk 1 above)
Comment (jakopako) [reply croncert-config.yml:738]: Extracting the date from the main page also gives us the advantage of being able to include _all_ events even if there are multiple events on a single day.

Comment (MarkJaroski) [reply croncert-config.yml:738]: I tried that, but for some reason it didn't seem to work.

Comment (jakopako) [reply croncert-config.yml:738]: hmmm that's strange

Comment (MarkJaroski) [reply croncert-config.yml:738]: Probably I missed some tiny thing. It looked like I wasn't getting a regex match in the URL with [0-9]{8}. Maybe I should have used a more precise expression.

**Pull request merged**